general/restrict-hostnames/examples/constraint.yaml

```
@@ -12,3 +12,4 @@ spec:
   parameters:
     exemptions:
       - '*.example.ca'
+    errorMsgAdditionalDetails: ""
```

Just gonna add a bit of documentation.
```suggestion
    # Hostnames that should be exempt from the policy. Glob patterns can be specified.
    exemptions:
      - '*.example.ca'
    # Extra information that should be appended to the error message.
    errorMsgAdditionalDetails: ""
```